packages/langium/src/grammar/langium-grammar-validator.ts

```diff
@@ -71,16 +71,14 @@ export class LangiumGrammarValidator {
         }
     }
 
-    checkFirstGrammarRule(grammar: ast.Grammar, accept: ValidationAcceptor): void {
-        const firstRule = getEntryRule(grammar);
-        if (firstRule) {
-            if (isDataTypeRule(firstRule)) {
-                accept('error', 'The entry rule cannot be a data type rule.', { node: firstRule, property: 'name' });
-            } else if (firstRule.fragment) {
-                accept('error', 'The entry rule cannot be a fragment.', { node: firstRule, property: 'name' });
-            }
-        } else {
+    checkEntryGrammarRule(grammar: ast.Grammar, accept: ValidationAcceptor): void {
```

Comment:
Suggestion: Please add another validation case here for fragment rules. A fragment rule can't be an entry rule.

Reply:
The grammar was changed such way, that it's impossible to create a rule with `fragment` and `entry` words simultaneously, so, I think such validation for fragment rules is redundant.

Reply:
I see, alright.